testing/run

```diff
@@ -64,8 +64,11 @@ prechecks() {
         echo "kubectl not found, symlinking to oc"
         mkdir -p bin
         cd bin
-        ln -s $(which oc) kubectl
+        if [[ ! -f kubectl ]]; then
+            ln -sf $(which oc) kubectl
```

Comment:
Use long form `--symbolic --force`

Reply:
ack,
and I removed the `--force` (the original idea) as testing `[[ ! -f kubectl ]]` makes it useless